airflow/models.py

```diff
@@ -805,6 +806,9 @@ def get_hook(self):
             elif self.conn_type == 'azure_data_lake':
                 from airflow.contrib.hooks.azure_data_lake_hook import AzureDataLakeHook
                 return AzureDataLakeHook(azure_data_lake_conn_id=self.conn_id)
+            elif self.conn_type == 'azure_container_instances':
+                from airflow.contrib.hooks.azure_data_lake_hook import AzureContainerHook
```

Comment:
Where is the `AzureContainerHook`?

Reply:
Good catch, was not updated in last iteration, will fix and do some more testing to find additional issues like this